doc/api/fs.md

```diff
@@ -1608,19 +1608,19 @@ On Linux, positional writes don't work when the file is opened in append mode.
 The kernel ignores the position argument and always appends the data to
 the end of the file.
 
-## fs.write(fd, data[, position[, encoding]], callback)
+## fs.write(fd, string[, position[, encoding]], callback)
```

Comment:
Is this syntax correct? Is it really impossible to specify encoding unless you also specify position? So, to write using an encoding at the current fd position, I would have to do `fs.write(fd, string, nil, encoding)`, specifying a position that has typeof != number?